packages/titanium-single-select/src/titanium-single-select.ts

```diff
@@ -384,6 +394,45 @@ export class TitaniumSingleSelectElement extends LitElement {
     [hidden] {
       display: none !important;
     }
+
+    @media (max-width: 768px) {
+      :host([open]:not([disableFullScreen])) {
+        max-width: 100% !important;
+        margin: 0 !important;
+        position: fixed;
+        z-index: 1000;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        background: #fff;
+      }
+
+      :host([open]:not([disableFullScreen])) input,
+      :host([open]:not([disableFullScreen])) input:hover {
+        position: fixed;
```

Comment:
This is for IOS, and it drove me crazy!!!!   On IOS when the keyboard comes out it does not resize the viewport, but rather pushes the viewport up to accommodate the keyboard. This would make the input always show up off screen, requiring the user to scroll up to get to it. Putting Position fixed helps IOS know where the input is so that it will scroll the window to the input accordingly.  The negative of this being that the window will now be scrolled to the top of the page when the user tries to go back to the form.  I believe for most of our use cases this will not be a problem, but it could be for some.  I tried scrolling the window back to the position it was at when the single-select was opened, but IOS doesn't respect that either.  Not sure of a better solution for this at this time.